Makefile

```
@@ -36,3 +36,9 @@ devtools:
 	env GOBIN= go install ./cmd/abigen
 	@type "solc" 2> /dev/null || echo 'Please install solc'
 	@type "protoc" 2> /dev/null || echo 'Please install protoc'
+
+forkdiff:
+	docker run \
```

```suggestion
	docker run --rm \
```

It's nice to clean up the docker container straight away when running it many times.

---

Good idea! I changed it.